error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/LexicographicalAnnotationAttributeListing.java

```diff
@@ -203,8 +203,10 @@ private ImmutableList<String> tokenize(Tree node) {
         /*
          * Tokens are split on `=` so that e.g. inline Spring property declarations are properly
          * sorted by key, then value.
+         * Leading and trailing double quotes are ignored for sorting purposes.
          */
-        return ImmutableList.copyOf(SourceCode.treeToString(node, state).split("=", -1));
+        return ImmutableList.copyOf(
+            SourceCode.treeToString(node, state).replaceAll("^\"|\"$", "").split("=", -1));
```

Comment:
Not sure if the right fix tbh.
But after loads of testing, I finally got that the escaped quotes were considered in the sorting, which we actually don't want, and we want to sort what's inside the string. 
Thus filtering out the leading and trailing quotes (if exists) from there.

Reply:
Thanks for tracking down the source of the odd sorting! It looks like we should use `ASTHelpers#constValue` here; I'll push a proposal :)

Reply:
Oh, didn't know about this one :D 
Thanks @Stephan202 for pushing those extras!